tensorboard/BUILD

```diff
@@ -19,11 +19,12 @@ package_group(
 py_binary(
     name = "tensorboard",
     srcs = ["main.py"],
+    data = ["webfiles.zip"],
```

Comment:
So I agree this is where the data dep belongs, but right now the code in `main.py` actually reads it from `program.get_default_assets_zip_provider()`.  It used to live in `default` but was moved to program back in #1631 (not really sure why, at this point).

IMO better at this point to just inline it into `main.py` and remove the fallback logic in `program.TensorBoard()` that calls it, so that `assets_zip_provider` must be specified by the caller.  That way we don't have skew between where the data dep is defined in the BUILD file, and where the code actually expects it to exist.  WDYT?

Reply:
Makes sense. Done.